### PR TITLE
Show iframe for Amazon listing

### PR DESCRIPTION
**Pull request by RoyEJohnson**
For: https://github.com/openstax/openstax.org/issues/2365

### CODE REVIEW

File: test/src/pages/details/get-this-title/get-this-title.test.js [hunk 1]
```diff
@@ -34,28 +29,28 @@ async function expectOptions(value) {
     await user.click(expandLink);
 }
 
-test('handles hiding and expanding non-preferred formats', async () => {
-    render(<GTTinContext />);
+// test('handles hiding and expanding non-preferred formats', async () => {
```
Comment (bethshook) [inline test/src/pages/details/get-this-title/get-this-title.test.js:32]: maybe a comment explaining the commented spec?

File: test/src/data/amazon-blurb.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default [
+    {
+        amazon_book_blurb: 'Support OpenStax with your purchase through Amazon.'
```
Comment (bethshook) [inline test/src/data/amazon-blurb.js:3]: just flagging that there is additional blurb text on [the card](https://github.com/openstax/openstax.org/issues/2365)

File: src/app/pages/details/common/get-this-title-files/order-print-copy/use-amazon-iframe.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    amazonDataLink: ReturnType<typeof useAmazonAssociatesLink>,
+    formatMessage: IntlShape['formatMessage']
+) {
+    const individual = formatMessage({
+        id: 'printcopy.individual',
+        defaultMessage: 'Individual'
+    });
+    const disclosure = formatMessage({
+        id: 'printcopy.disclosure',
+        defaultMessage: '***'
```
Comment (bethshook) [inline src/app/pages/details/common/get-this-title-files/order-print-copy/use-amazon-iframe.ts:35]: can you not just set this to `amazonDataLink.disclosure`?

**Review by bethshook — APPROVED** [review]

File: src/app/pages/details/common/get-this-title-files/order-print-copy/use-amazon-iframe.ts (hunk 3 above)
Comment (RoyEJohnson) [reply src/app/pages/details/common/get-this-title-files/order-print-copy/use-amazon-iframe.ts:35]: There's a little trickery here to handle Spanish. the amazonDataLink.disclosure is in English.

**Pull request merged**